JS/Anna_G/TaskOfBookStore/src/index.js

```diff
@@ -0,0 +1,65 @@
+import _ from 'lodash';
+import data from '../resources/books.json'
+
+const formBlock = document.querySelector('#form-block');
+const book = document.querySelector('#bookdiv');
+const globalDiv = document.getElementsByClassName('a');
+
+function createElements(type, newClassName) {
+    const variableName = document.createElement(type);
+    variableName.className = newClassName;
+    return variableName;
+}
+
+function initializedBookElements(index, bookElements){
+    bookElements.titleH.textContent = data[index].title;
+    bookElements.authorH.textContent = data[index].author;
+    bookElements.elem.src = data[index].imageLink;
+    bookElements.button.innerHTML = "Delete";
+}
+
+function appendBook(book, index) {
+    const bookElements = createBookElements();
+    const column = createElements('div', 'column');
+    initializedBookElements(index,bookElements);
+    let key;
+    for (key in bookElements) {
+        column.appendChild(bookElements[key])
+        console.log(key);
+    }
+    book.appendChild(column);
+}
+
+function createBookElements() {
+    return {
+        titleH: createElements('h1', 'title'),
+        authorH: createElements('h2', 'author'),
+        elem: createElements('img', 'img'),
+        button: createElements('button', 'delete')
+    }
+}
+
+
+formBlock.addEventListener('submit', function (event) {
+    event.preventDefault();
+    book.innerHTML = "";
+    const inputValue = document.querySelector('#input-block').value;
+    if (inputValue > 0 && inputValue < 20) {
+        document.querySelector("#input-block").value = "";
```

Comment:
you are doing this empty assignment in both cases inside/outside of if. Improve the code to not do it multiple times